### DISCUSSION TITLE
Удалить софт

### DISCUSSION THREAD

**Discussion by tomcattack** [Anonym-tsk/nfqws-keenetic#131] [Q&A] [answered]
Подскажите, как корректно удалить софт из памяти роутера, чтобы потом установить на USB-флешку.

**Comment by alexxxsubbota** [comment, marked as answer]
Форматирование встроенного накопителя.
1) Управление - OPKG-накопитель - меняете на "не выбран", жмете сохранить.
2) Управление-приложения - втсроенное хранилище - жамкаете кнопку отключить
3) Теперь в cli роутера(я захожу через putty на 192.168.1.1 порт 22) выполняете команду: "erase storage:"

на этом все, внутренний накопитель отформатирован, можно подключить его снова (см пункт №2), можно установить подсистему Linux с нуля

Если нужно лишь установить на другой(внешний) носиттель можно обойтись пунктом №1, при этом выбрать целевой накопитель для установки подсистемы linux, предварительно создав на нем дирикторию install и поместив в нее архив для установки. Удалять и форматировать внутренний накопитель не обязательно.

**Reply by tomcattack** [reply]
Спасибо
